spring-graphql/src/main/java/org/springframework/graphql/data/query/QueryByExampleDataFetcher.java

```diff
@@ -304,6 +325,23 @@ public DataFetcher<Iterable<R>> many() {
 
 	}
 
+	/**
+	 * Callback interface that can be used to customize QueryByExampleDataFetcher {@link Builder} to
+	 * change its configuration.
```

Comment:
Maybe we could mention here, as well as on all similar interfaces, that at present this is supported by the auto-registration methods which check if a repository implements this interface.

Reply:
Good point, will update the code accordingly.